etc/eks-env.template

```diff
@@ -2,48 +2,38 @@
 # Common Development and Distribution License (CDDL) that can be found in the LICENSE file
 # Copy this template to eks-env.cfg and adjust for your environment
 
+#########################################################################
+### POPULATE THE FOLLOWING VALUES PRIOR TO RUNNING ANY OF THE SCRIPTS ###
+#########################################################################
 
-# CloudFormation Stack Name
-EKS_STACK_NAME="cluster1"
-
-# EKS Cluster Name
-EKS_CLUSTER_NAME="cluster1"
-
-# Kubernetes Cluster Version
-# To get specific cluster versions you can run the command
-EKS_CLUSTER_VERSION="latest"
-
-# EKS Service Role Arn
-EKS_ROLE_ARN="arn:aws:iam::1111111:role/eksServiceRole"
+#### ** ENVIRONMENT SETUP ** ####
 
-# EKS VPC ID
-EKS_VPC_ID="vpc-11111"
+# EKS VPC stack name
+EKS_VPC_STACK_NAME="vpc1"
```

Comment:
Would it be meaningful to make the name of this VPC as infra-vpc so that it is distinguished from the other VPC that is created for worker nodes.

Reply:
Not sure what you mean here? There is only 1 VPC. Do you mean the stack name. I've changed the worker node stack name from EKS_STACK_NAME, which was confusing, to EKS_WORKER_NODE_STACK_NAME